tests/installation/ipxe_install.pm

```diff
@@ -141,50 +181,54 @@ END_BOOTSCRIPT
     diag "$response->{status} $response->{reason}\n";
 }
 
+sub enter_o3_ipxe_boot_entry {
+    ipmitool('chassis power reset') unless check_screen([qw(o3-ipxe-menu ipxe-boot-failure)], 180);
+    assert_screen('o3-ipxe-menu', 210);
+    my $key = get_var('HOST_INSTALL_AUTOYAST') ? (is_kvm_host ? 'k' : 'z') : 't';
+    send_key "$key";
+    # try one more time as sometimes sending key does not take effect
+    send_key "$key" if check_screen('o3-ipxe-menu');
+    # confirm the dialog asking for user modifications to kernel, cmdline and initrd
+    # set limit to 10 in case some keys don't go through
+    for (1 .. 10) {
+        last if check_screen([qw(load-linux-kernel load-initrd)], 3);
+        send_key "ret";
+    }
+}
+
 
 sub run {
     my $self = shift;
 
     poweroff_host;
 
     #virtualization tests use a static ipxe configuration file in O3
-    set_bootscript unless get_var('VIRT_AUTOTEST') && is_tumbleweed;
+    set_bootscript unless get_var('VIRT_AUTOTEST') and is_tumbleweed and get_var('OPENQA_URL') =~ "openqa.opensuse.org";
```

Comment:
According to your comment, is_tumbleweed should be removed from ```unless``` ?

Reply:
I don't know if ALP behaves same way as Tumbleweed, so just to be limited to TW.

Reply:
This should be `set_bootscript unless get_var('IPXE_STATIC'); or something like that. Then set `IPXE_STATIC` whereever it's needed. Checking `DISTRI` or `OPENQA_URL` is really terrible approach.

Reply:
And please always keep in mind the baremetal support service has no persistency for it's data. If, for any reason, the service restarts, it will lose all the static configuration. Would be a better solution to always set your bootscript if you don't want to risk boot failures in that case.

Reply:
> And please always keep in mind the baremetal support service has no persistency for it's data. If, for any reason, the service restarts, it will lose all the static configuration. Would be a better solution to always set your bootscript if you don't want to risk boot failures in that case.

Thanks. There is no baremetal support service deployed or be accessible on O3. There is only a static iPXE menu shared by tests.

Reply:
Okay! If you need the service on o3 just reach out, I bet we can do something with that.

Reply:
Thank you, Michael. I did some research before my tests are deployed in O3 with iPXE. The dhcp server in O3 is dnsmasq, it seems that the http redirection for designated machines are not supported. All the tests have to share the same iPXE configurations. In addition, accessing internal SUSE network(IP likes 10.xx) is not allowed on O3. So finally we choosed the static iPXE menu.

Reply:
> This should be `set_bootscript unless get_var('IPXE_STATIC');` or something like that. Then set `IPXE_STATIC` whereever it's needed. Checking `DISTRI` or `OPENQA_URL` is really terrible approach.

I noticed that OSD began to use the same static iPXE menu on O3. `get_var('IPXE_STATIC')` is more approriate. thanks.